lib/widgets/theme.dart

```diff
@@ -82,10 +89,7 @@ ThemeData zulipThemeData(BuildContext context) {
     //   https://api.flutter.dev/flutter/material/ColorScheme/ColorScheme.fromSeed.html
     // Or try this tool to see the whole palette:
     //   https://m3.material.io/theme-builder#/custom
-    colorScheme: ColorScheme.fromSeed(
-      brightness: brightness,
-      seedColor: kZulipBrandColor,
-    ),
+    colorScheme: colorScheme,
```

Comment:
nit: the comment above here is about the construction of the ColorScheme, so should move along with that constructor call